code/modules/mob/living/simple_animal/borer/borer_powers.dm

```diff
@@ -205,7 +205,7 @@
 		to_chat(src, SPAN_WARNING("You don't have enough chemicals!"))
 		return
 
-	var/list/choices = list("Inaprovaline" = /datum/reagent/inaprovaline, "Bicaridine" = /datum/reagent/bicaridine, "Kelotane" = /datum/reagent/kelotane, "Dylovene" = /datum/reagent/dylovene, "Hyperzine" = /datum/reagent/hyperzine, "Peridaxon" = /datum/reagent/peridaxon, "Mortaphenyl" = /datum/reagent/mortaphenyl, "Neurostabin" = /datum/reagent/mental/neurostabin)
+	var/list/choices = list("Inaprovaline" = /datum/reagent/inaprovaline, "Bicaridine" = /datum/reagent/bicaridine, "Kelotane" = /datum/reagent/kelotane, "Dylovene" = /datum/reagent/dylovene, "Hyperzine" = /datum/reagent/hyperzine, "Peridaxon" = /datum/reagent/peridaxon, "Mortaphenyl" = /datum/reagent/mortaphenyl, "Neurapan" = /datum/reagent/mental/neurapan) //Switched out Neurostabin for Neurapan which may be more interesting, also dangerous if a borer injects too much.
```

Comment:
These comments you're placing are helpful now, but they should either be in the PR description or the changelog. They'll make no sense to whoever goes back to revisit this stuff later.

Reply:
Removed the unnecessary ones and made the more necessary ones a tad clearer. I've made a bad habit of leaving comments everywhere, regardless of necessity and them being clear enough or not.